src/components/rows/MainRow2/index.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import PropTypes from "prop-types";
+import SliderMain from "../../atoms/SliderMain";
```

Comment:
let's group the third party imports together ,the same repo imports together and css together

Reply:
done